src/@next/Banner/Banner.tsx

```diff
@@ -53,43 +64,87 @@ export const Banner = React.forwardRef<HTMLDivElement, BannerProps>(
       ? iconName
       : iconByStatus || iconNameStatusMap['info'];
 
+    if (type === 'static')
+      return (
+        <StyledBanner
+          ref={ref}
+          data-titled={!!title}
+          data-status={status}
+          {...props}
+        >
+          <StyledBannerTitleContainer>
+            <Icon name={iconNameValue} />
+            {title && (
+              <StyledBannerTitle>
+                <Typography as="span" variant="subtitle1">
+                  {title}
+                </Typography>
+              </StyledBannerTitle>
+            )}
+            {dismissable && (
+              <StyledCloseIconWrapper
+                role="button"
+                onClick={() => onDismiss?.()}
+              >
+                <Icon name="ri-close" />
+              </StyledCloseIconWrapper>
+            )}
+          </StyledBannerTitleContainer>
+          <StyledBannerContentContainer>
+            <Typography as="div" variant="body1">
+              {children}
+            </Typography>
+          </StyledBannerContentContainer>
+          <StyledBannerContentContainer>
+            {action &&
```

Comment:
Is this not working? https://github.com/glints-dev/glints-aries/pull/838#discussion_r1209072975

Reply:
oh the problem with that is I need to check if action is given but secondaryAction is not given, I'll just display `{action}` without the buttonGroup, so it is not working as I intended

Is the current code not readable or not clean enough?
The current logic that I intended is:
- If action is not undefined:
- - If secondaryAction is defined: show button group with both actions
- - Else: show only action

Reply:
Any reason why you need to get rid of button group when only action present? Is it breaking the style?

Reply:
Let me check soon, was concern that there might me some additional unintended spacings if I use button group... but I think you're right that we can still use the button group even there is only one button, okay will change this soon

Reply:
I'll just make it like this, I hope it's readable enough now:
```
    const actionComponent = (
      <>
        {action && (
          <ButtonGroup>
            {action}
            {secondaryAction && secondaryAction}
          </ButtonGroup>
        )}
      </>
    );
```